src/gleam/float.gleam

```diff
@@ -433,6 +433,55 @@ fn do_product(numbers: List(Float), initial: Float) -> Float {
 @external(javascript, "../gleam_stdlib.mjs", "random_uniform")
 pub fn random() -> Float
 
+/// Computes the modulo of an float division of inputs as a `Result`.
+///
+/// Returns division of the inputs as a `Result`: If the given divisor equals
+/// `0`, this function returns an `Error`.
+///
+/// ## Examples
+///
+/// ```gleam
+/// modulo(3.0, 2.0)
+/// // -> Ok(1.0)
+/// ```
+///
+/// ```gleam
+/// modulo(1.0, 0.0)
+/// // -> Error(Nil)
+/// ```
+///
+/// ```gleam
+/// modulo(10.0, -1.0)
+/// // -> Ok(0.0)
+/// ```
+///
+/// ```gleam
+/// modulo(13.0, by: 3.0)
+/// // -> Ok(1.0)
+/// ```
+///
+/// ```gleam
+/// modulo(-13.0, by: 3.0)
+/// // -> Ok(2.0)
+/// ```
+///
+/// ```gleam
+/// modulo(13.0, by: -3.0)
+/// // -> Ok(-2.0)
+/// ```
+///
+/// ```gleam
+/// modulo(-13.0, by: -3.0)
+/// // -> Ok(-1.0)
+/// ```
```

Comment:
Can we have fewer examples please? It's a little overwhelming I think

Reply:
The comments were copied from the `int.modulo` function, they explain all the possible behaviours.

- When divisor is non-zero and when it is zero.
- That the modulo follows the sign of the divisor.

Can we keep the examples?

Reply:
There is too much information here, please reduce it. The examples section is not suitable for an exhaustive description of the behaviour.